src/appshell/view/appmenumodel.cpp

```diff
@@ -42,24 +43,17 @@ void AppMenuModel::load()
             << toolsItem()
             << helpItem();
 
-    globalContext()->currentMasterNotationChanged().onNotify(this, [this]() {
-        load();
-
-        currentMasterNotation()->notation()->notationChanged().onNotify(this, [this]() {
-            load();
-        });
-
-        currentMasterNotation()->notation()->interaction()->selectionChanged().onNotify(this, [this]() {
-            load();
-        });
-    });
+    setupConnections();
 
     emit itemsChanged();
 }
 
-void AppMenuModel::handleAction(const QString& actionCode)
+void AppMenuModel::handleAction(const QString& actionCode, int actionIndex)
 {
-    actionsDispatcher()->dispatch(codeFromQString(actionCode));
+    ActionCode _actionCode = codeFromQString(actionCode);
```

Comment:
I believe, better in such a case  use `val_` for args

Reply:
renamed